runelite-client/src/main/java/net/runelite/client/plugins/fps/FpsOverlay.java

```diff
@@ -111,9 +111,8 @@ public Dimension render(Graphics2D graphics)
 		if (fps < FPS_SIZE)
 		{
 			final int width = client.getCanvas().getWidth();
-			final Point point = new Point(width - VALUE_X_OFFSET, Y_OFFSET);
-
-			OverlayUtil.renderTextLocation(graphics, point, fpsNums[fps], getFpsValueColor());
+			final Point point = new Point(width - VALUE_X_OFFSET - graphics.getFontMetrics().stringWidth("fps"), Y_OFFSET);			
```

Comment:
Reuse the word "fps" also, it should be uppercase and have space before it

Reply:
Where do you want to reuse the word FPS? It only gets used once

Reply:
You used it twice, for getting it's width and then for drawing it

Reply:
Oh lmao. That's true, my bad.